refactor setup image, upgrade lighttpd, and improve README (#9)
PDOK/lighttpd-docker: README.md

```docker
docker build -t pdok/lighttpd .
docker run --rm -v `pwd`/www/:/var/www -p 80:80 --name lighttpd-example pdok/lighttpd
docker stop lighttpd-example
```

## Introduction

> This project aims to fulfill the need in creating a [lighttpd](https://www.lighttpd.net/) container that is deployable on a scalable infrastructure, like [Kubernetes](https://kubernetes.io/). This means that this images is based on a lightweight base-image.

Reviewer: ```suggestion
This project aims to fulfill the need in creating a [lighttpd](https://www.lighttpd.net/) container that is deployable on a scalable infrastructure, like [Kubernetes](https://kubernetes.io/). This means that this image is based on a lightweight base-image.

```

> The image comes with a default, overridable [`lighttdp.conf`](config/lighttpd.conf) file that is specically aimed at serving out static files for a webapplication.

Reviewer: ```suggestion
The image comes with a default, overridable [`lighttdp.conf`](config/lighttpd.conf) file that is specifically aimed at serving static files for a web application.

```

### Run

This image can be run straight from the commandline. Both the default configuration file and the default asset directory can be overridden from the commandline:

- override default configuration file by mounting a directory with a `lighttpd.conf` file at `/srv/lighttpd` in the container
- override default asset directory by mounting a directory with assets to serve at `/var/www` in the container

```sh
docker run --rm -p 80:80 --name lighttpd-example -v `pwd`/config:/srv/lighttpd -v `pwd`/www:/var/www pdok/lighttpd
```

Running the example above will create a service on the url <http://localhost/>

The environment variables that can be set are the following:

- `DEBUG`
- `MIN_PROCS`
- `MAX_PROCS`
- `MAX_LOAD_PER_PROC`
- `IDLE_TIMEOUT`

> The environment variable have a default value set in the Dockerfile.

Reviewer: ```suggestion
The environment variables have a default value set in the Dockerfile.

```

MAX_PROCS
MAX_LOAD_PER_PROC
IDLE_TIMEOUT
```
- `DEBUG`
- `MIN_PROCS`
- `MAX_PROCS`
- `MAX_LOAD_PER_PROC`
- `IDLE_TIMEOUT`

The ENV variables have a default value set in the Dockerfile.
The environment variables have a default value set in the Dockerfile.

## Misc
### Use image in another Docker image

### How to Contribute
See below example for how to use this image to build another image to serve out static files:

Make a pull request...
```sh
work_dir="/tmp/$(uuidgen)"
mkdir "$work_dir"
cd "$work_dir"
echo "<h1>TEST</h1>" > index.html

### Contact
cat > Dockerfile << EOF
FROM pdok/lighttpd
COPY index.html /var/www/index.html
EOF

> Contacting the maintainers can be done through the issue tracker.

Reviewer 2: Why remove the 'How to contribute'
